src/template-tags/tickets.php

```diff
@@ -199,10 +199,26 @@ function tribe_events_ticket_is_on_sale( Tribe__Tickets__Ticket_Object $ticket )
 	 * @return string
 	 */
 	function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $ticket ) {
-		$stock     = $ticket->stock();
-		$sold      = $ticket->qty_sold();
-		$cancelled = $ticket->qty_cancelled();
-		$pending   = $ticket->qty_pending();
+		$stock       = $ticket->stock();
+		$sold        = $ticket->qty_sold();
+		$cancelled   = $ticket->qty_cancelled();
+		$pending     = $ticket->qty_pending();
+		$woo_tickets = Tribe__Tickets_Plus__Commerce__WooCommerce__Main::get_instance();
```

Comment:
WooCommerce pieces should probably live in `event-tickets-plus` since there may be installs where ET is activated but ET+ is not.